City-of-Pori/pori-kada
# PORI-326 Added news archive link to front page and visitpori front pa…

…ge, cleaned up the theme just a bit

To test:

1. drush fra -y && drush cc all
2. Go to front page and make sure there is News archive link
- When screen size is more than 1320px wide on the top-right corner of current flow block
- When screen size is smaller than 1320px wide just after the current flow block
3. Check the same thing from visitpori front page.
